set default values for openfaas parameters (#216)
falcosecurity/charts: falcosidekick/CHANGELOG.md

Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).

## 0.3.3

### Fixes

> * Set default values for OpenFaaS output type parameters
> * Fixes on documentations

Reviewer: ```suggestion
* Set default values for `OpenFaaS` output type parameters
* Fixes of documentation
```

Author: done

> * Set default values for OpenFaaS output type parameters
> * Fixes of documentations

Reviewer: ```suggestion
* Set default values for `OpenFaaS` output type parameters
* Fixes of documentation
```

Author: ok, now its done 👍
